homeassistant/components/glances/config_flow.py

```diff
@@ -37,6 +39,7 @@
         vol.Required(CONF_VERSION, default=DEFAULT_VERSION): int,
         vol.Optional(CONF_SSL, default=False): bool,
         vol.Optional(CONF_VERIFY_SSL, default=False): bool,
+        vol.Optional(CONF_DOCKER_CONTAINERS, default=DEFAULT_DOCKER_CONTAINERS): list,
```

Comment:
Optional options should go under config flow options flow.

Reply:
Is there even an option to configure lists in config flow? When I open the config flow for my integration I get this error:
`
Traceback (most recent call last):
  File "/home/adrian/home-assistant/venv/lib/python3.6/site-packages/aiohttp/web_protocol.py", line 418, in start
    resp = await task
  File "/home/adrian/home-assistant/venv/lib/python3.6/site-packages/aiohttp/web_app.py", line 458, in _handle
    resp = await handler(request)
  File "/home/adrian/home-assistant/venv/lib/python3.6/site-packages/aiohttp/web_middlewares.py", line 119, in impl
    return await handler(request)
  File "/home/adrian/home-assistant/homeassistant/components/http/real_ip.py", line 40, in real_ip_middleware
    return await handler(request)
  File "/home/adrian/home-assistant/homeassistant/components/http/ban.py", line 73, in ban_middleware
    return await handler(request)
  File "/home/adrian/home-assistant/homeassistant/components/http/auth.py", line 136, in auth_middleware
    return await handler(request)
  File "/home/adrian/home-assistant/homeassistant/components/http/view.py", line 122, in handle
    result = await result
  File "/home/adrian/home-assistant/homeassistant/components/config/config_entries.py", line 130, in post
    return await super().post(request)
  File "/home/adrian/home-assistant/homeassistant/components/http/data_validator.py", line 48, in wrapper
    result = await method(view, request, *args, **kwargs)
  File "/home/adrian/home-assistant/homeassistant/helpers/data_entry_flow.py", line 67, in post
    result = self._prepare_result_json(result)
  File "/home/adrian/home-assistant/homeassistant/components/config/config_entries.py", line 135, in _prepare_result_json
    return super()._prepare_result_json(result)
  File "/home/adrian/home-assistant/homeassistant/helpers/data_entry_flow.py", line 40, in _prepare_result_json
    data["data_schema"] = voluptuous_serialize.convert(schema)
  File "/home/adrian/home-assistant/venv/lib/python3.6/site-packages/voluptuous_serialize/__init__.py", line 38, in convert
    pval = convert(value, custom_serializer=custom_serializer)
  File "/home/adrian/home-assistant/venv/lib/python3.6/site-packages/voluptuous_serialize/__init__.py", line 108, in convert
    raise ValueError('Unable to convert schema: {}'.format(schema))
ValueError: Unable to convert schema: <class 'list'>
`